src/plugins/world_control/WorldControl.hh

```diff
@@ -54,15 +54,19 @@ namespace plugins
   ///
   /// ## Configuration
   ///
-  /// * \<play_pause\> : Set to true to see a play/pause and step buttons,
+  /// * \<play_pause\> : Set to true to see a play/pause button,
   ///                    false by default.
+  /// * \<step\> : Set to true to see a step button, false by default.
```

Comment:
Should we update the defaults be `true` as well?

Reply:
I tried to do this in a way that the defaults wouldn't need to be changed, because that would break existing behaviour. For example, a user who currently has only `<play_pause>` will continue not getting the step button, because `<step>` is false by default.

The only situation that changed is when the `<plugin>` tag is completely empty and has the effect of flipping all defaults to true. This is technically a change in behaviour, but I think that wouldn't break any existing users, because no one should be using the plugin completely empty, and it would fix a usability bug.

Let me know if the reasoning sounds ok to you

Reply:
Ah ok, thanks for explaining. Sounds good to me